libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -251,10 +265,32 @@ export const appStrings = {
     </UiString>
   ),
 
+  noteBmdCastingBallot: () => (
+    <UiString uiStringKey="noteBmdCastingBallot">Casting Ballot...</UiString>
+  ),
+
   noteClearingBallot: () => (
```

Comment:
Probably want to call this one `noteBmdClearingBallot` for consistency (unless I'm forgetting about a non-BMD usage of it!)

Reply:
good catch - missed that